tests/test_modalities/test_ecephys/test_ecephys_interfaces.py

```diff
@@ -110,6 +110,34 @@ def test_electrode_indices(self, setup_interface):
             electrode_table_region_properties = electrode_table_region["property"].to_list()
             assert electrode_table_region_properties == property
 
+    def test_rename_unit_ids(self):
+        interface = MockSortingInterface(num_units=3)
+
+        # Rename all units
+        unit_ids_map = {"0": "neuron_1", "1": "neuron_2", "2": "neuron_3"}
+        interface.rename_unit_ids(unit_ids_map)
```

Comment:
Would it be worth adding a test case when `unit_ids_map` values are not strings?

Reply:
Thanks for the explanation, makes sense now.